hooks/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+import subprocess
+
+def check_fantomas():
+    result = subprocess.run(["dotnet", "tool", "run", "fantomas", "--check", "-r", "."])
+    if result.returncode != 0:
+        print(result.stdout)
```

Comment:
Haven't tested, but doesn't subprocess.run print output by default anyway?

Reply:
Huh, you are quite right. I'd never noticed the sneaky line "No changes required" that gets printed out.